zulipterminal/themes/gruvbox_dark.py

```diff
@@ -19,10 +19,10 @@
 STYLES = {
     # style_name       :  foreground                   background
     None               : (Color.LIGHT2,                Color.DARK0_HARD),
-    'selected'         : (Color.LIGHT2,                Color.FADED_BLUE),
-    'msg_selected'     : (Color.LIGHT2,                Color.FADED_BLUE),
-    'header'           : (Color.NEUTRAL_BLUE,          Color.FADED_BLUE),
-    'general_narrow'   : (Color.LIGHT2,                Color.FADED_BLUE),
+    'selected'         : (Color.DARK0_HARD,            Color.NEUTRAL_BLUE),
```

Comment:
From reading the blame, there seems to be a reason why the foreground is light. See https://github.com/zulip/zulip-terminal/commit/0800dfa12a50430a3265f2e98f2106b52b269efa.

Reply:
I believe we should stick to intended palette colors over matching zt_dark, and syntax highlighting mitigates the other reason that change was made?

Reply:
It might have looked better with white for ZT dark as well?

Reply:
This is what is in zt_dark.py: `    'selected'        : (Color.WHITE,               Color.DARK_BLUE),`. Making it `DARK0_HARD` actually is not making it consistent.

Reply:
@rht I agree it is not consistent.

I experimented with `LIGHT2` with the different blues which should be in gruvbox-dark (as backgrounds), and the contrast wasn't great, which is why I selected `DARK0_HARD`.

For me, the blue used in zt_dark is darker, so white text works better there.

Your terminals and opinions may differ, so please do follow-up :)

Reply:
OK, I'm good with the `DARK0_HARD` then. It will look the same in my terminal, because 24 bit is true color.

Reply:
@rht To confirm, you're OK with the set of dark theme changes?

Reply:
Yes.